shared/hack/install.sh

```diff
@@ -145,6 +145,11 @@ install_checkov() {
     checkov --version
 }
 
+install_go(){
+    curl "${CURL_OPTS[@]}" -o "$TMPDIR/go.tgz" https://go.dev/dl/go1.19.1.linux-amd64.tar.gz
```

Comment:
it should use `GO_VERSION` variable to represent go version, for example `https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz`